Pending_updates/cit_031_air_quality_PM25_concentration/README.md

```diff
@@ -0,0 +1,12 @@
+## Air Quality: Surface Fine Particulate Matter (PM2.5) Concentrations Dataset Pre-processing
+This file describes the data pre-processing that was done to [the Annual averaged 0.01° × 0.01° w GWR adjustment PM2.5](http://fizz.phys.dal.ca/~atmos/martin/?page_id=140) for [display on Resource Watch](https://resourcewatch.org/data/explore/6e6750da-50c8-4b52-914f-b0d663a7ab59).
+
+Each year of data is provided by the source as a NetCDF file. Before we uploaded each data file to Google Earth Engine, we converted its format from NetCDF to GeoTIFF.
```

Comment:
Any time we are converting a netcdf to a tiff, make sure to name the variables we pulled to show on RW
```suggestion
Each year of data is provided by the source as a NetCDF file. In order to display this data on Resource Watch, the PM25 variable in each NetCDF was converted to a GeoTIFF, and the latitude and longitude fields had to be switched because they were transposed in the original dataset.
```